processes/runPost.R

```diff
@@ -70,8 +78,8 @@ for(i in 1:length(flLst[[1]])){
             boxnames=boxplot_these, rpnames=rp_these, trajnames=traj_these)
 }
 
-# Output the management procedures text file in the figure directory
```

Comment:
Reading in mproc has been moved to earlier and I switched this to just read.csv().